Graphics/Win32/Message.hsc

```diff
@@ -158,6 +158,7 @@ type WindowMessage   = DWORD
  , wM_QUEUESYNC         = WM_QUEUESYNC
  , wM_USER              = WM_USER
  , wM_APP               = WM_APP
+ , wM_SETICON           = WM_SETICON
```

Comment:
Is there any rationale to the ordering here?

Reply:
Not that I'm aware off, they were added from an import from another project and I didn't clean up all the code :)

In principle they should be alphabetical but I'm not that bothered